packages/rpc/src/index.ts

```diff
@@ -87,7 +89,7 @@ class RpcProxy {
     }
     const header: Header = await this.rpc.get_tip_header();
     const blockNumber = BI.from(header.number);
-    while (true) {
+    for (;;) {
```

Comment:
why refactor here? some conventions?

Reply:
`while(true)` will trigger eslint error 'no-constant-condition'

Reply:
Okay, looks reasonable, but I would say that `while (true)` is more intuitive.

Can we change the rule in `.eslintrc.js` to `no-constant-condition: ["error", { "checkLoops": false }]` here? @homura 

Reference: https://eslint.org/docs/rules/no-constant-condition#checkloops